CascadeCrossoverがオプションで選択できない

`--crossover-type Cascade`にすると落ちる．
```
2021-01-20 12:12:03 [main] [ERROR] CUILauncher - "Cascade" is not a valid value for "--crossover-type"
java.lang.IllegalArgumentException: "Cascade" is not a valid value for "--crossover-type"
	at jp.kusumotolab.kgenprog.Configuration$Builder.createFromCmdLineArgs(Configuration.java:385)
	at jp.kusumotolab.kgenprog.Configuration$Builder.buildFromCmdLineArgs(Configuration.java:352)
	at jp.kusumotolab.kgenprog.CUILauncher.main(CUILauncher.java:30)
```

--- a/src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java
+++ b/src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java
@@ -55,7 +55,7 @@ public interface Crossover {
        * 交叉を行うインスタンスを生成するメソッド．
        *
        * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
-       *      SecondVariantSelectionStrategy, int)
+       *      SecondVariantSelectionStrategy, int, int)
        *
        * @param random 交叉処理の内部でランダム処理を行うためのシード
        * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム
@@ -84,7 +84,7 @@ public interface Crossover {
        * 交叉を行うインスタンスを生成するメソッド．
        *
        * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
-       *      SecondVariantSelectionStrategy, int)
+       *      SecondVariantSelectionStrategy, int, int)
        *
        * @param random 交叉処理の内部でランダム処理を行うためのシード
        * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム
@@ -113,7 +113,7 @@ public interface Crossover {
        * 交叉を行うインスタンスを生成するメソッド．
        *
        * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
-       *      SecondVariantSelectionStrategy, int)
+       *      SecondVariantSelectionStrategy, int, int)
        *
        * @param random 交叉処理の内部でランダム処理を行うためのシード
        * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム
@@ -142,7 +142,7 @@ public interface Crossover {
        * 交叉を行うインスタンスを生成するメソッド．
        *
        * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
-       *      SecondVariantSelectionStrategy, int)
+       *      SecondVariantSelectionStrategy, int, int)
        *
        * @param random 交叉処理の内部でランダム処理を行うためのシード
        * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム

--- a/src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java
+++ b/src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java
@@ -130,6 +130,35 @@ public interface Crossover {
         return new UniformCrossover(random, firstVariantSelectionStrategy,
             secondVariantSelectionStrategy, generatingCount, requiredSolutions);
       }
+    },
+
+    /**
+     * 直列交叉を表す型．
+     *
+     * @see jp.kusumotolab.kgenprog.ga.crossover.CascadeCrossover
+     */
+    Cascade {
+      /**
+       * 交叉を行うインスタンスを生成するメソッド．
+       *
+       * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
+       *      SecondVariantSelectionStrategy, int)
+       *
+       * @param random 交叉処理の内部でランダム処理を行うためのシード
+       * @param firstVariantSelectionStrategy 1つ目の親を選ぶためのアルゴリズム
+       * @param secondVariantSelectionStrategy 2つ目の親を選ぶためのアルゴリズム
+       * @param generatingCount 一世代の交叉処理で生成する個体の数
+       * @param requiredSolutions 生成する必要がある修正プログラムの数
+       * @return 交叉を行うインスタンス
+       */
+      @Override
+      public Crossover initialize(final Random random,
+          final FirstVariantSelectionStrategy firstVariantSelectionStrategy,
+          final SecondVariantSelectionStrategy secondVariantSelectionStrategy,
+          final int generatingCount, final int requiredSolutions) {
+        return new CascadeCrossover(firstVariantSelectionStrategy,
+            secondVariantSelectionStrategy, requiredSolutions);
+      }
     };
 
     /**

